docs/logs/search-through-the-sematext-api.md

```diff
@@ -29,15 +29,15 @@ When you use the API, here are the things you need to know:
 
 ## Searching
 
-Sematext supports a subset of Elasticsearch APIs, with rich query language and extensive capabilities of searching through data you've sent to it. The supported Search APIs include, but are not limited to:
+Sematext supports a subset of its own Search API —compatible with Elasticsearch and Opensearch—, with rich query language and extensive capabilities of searching through data you've sent to it. The supported Search APIs include, but are not limited to:
```

Comment:
Maybe instead:
Sematext API supports a subset of Elasticsearch / OpenSearch APIs, with ....